mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +55,104 @@ def resource_for_region(self, region, uri_prefix, session_store):
         Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
         implement :obj:`IAPIMock`.
         """
+        self.session_store = session_store
```

Comment:
The aforementioned test appears to be flakey.  I opened a separate mimic issue concerning it.